tests/cases/fourslash/server/ngProxy1.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path="../fourslash.ts"/>
+
+// @Filename: tsconfig.json
+//// {
+////     "compilerOptions": {
+////         "plugins": [
+////             { "name": "ng-proxy" }
+////         ]
+////     },
+////     "files": ["a.ts"]
+//// }
+
+// @Filename: a.ts
+//// let x = [1, 2];
+//// x/**/
+//// 
+
+goTo.marker();
+verify.quickInfoIs('Proxied x: number[]Check');

```

Comment:
The tests seem a little light for a new feature. We need to test what happens when the plugin fails to load (either throws on initialization or can't be found), if the plugin config is misconfigured, if the proxied API returns invalid results, etc... We need to ensure we're either resilient or give an actionable error to help identify the cause, else we'll be troubleshooting a lot of issues that aren't ours 😟